network/src/web.ts

```diff
@@ -0,0 +1,77 @@
+import { WebPlugin, PluginListenerHandle } from '@capacitor/core';
+import { NetworkPlugin, NetworkStatus } from './definitions';
+
+declare var window: any;
+
+export class NetworkWeb extends WebPlugin implements NetworkPlugin {
+  listenerFunction: any = null;
+
+  constructor() {
+    super({ name: 'Network' });
+  }
+
+  getStatus(): Promise<NetworkStatus> {
+    return new Promise((resolve, reject) => {
+      if (!window.navigator) {
+        reject('Network info not available');
+        return;
+      }
+
+      let connected = window.navigator.onLine;
```

Comment:
Prefer `const` vs `let` when the variable isn't reassigned.

```suggestion
      const connected = window.navigator.onLine;
```